src/lib/requestManager.ts

```diff
@@ -1,9 +1,46 @@
-import axios, { Method as HTTPMethod, ResponseType, AxiosRequestConfig, AxiosResponse } from 'axios';
+import axios, { Method as HTTPMethod, AxiosRequestConfig, AxiosResponse, AxiosTransformer } from 'axios';
+import { camelizeKeys } from 'humps';
+import _ from 'lodash';
```

Comment:
Should we just import the needed functions instead of the whole `lodash` package?
```suggestion
import { mapKeys, snakeCase } from 'lodash';
```